tiledb/sm/group/group.h

```diff
@@ -355,7 +346,7 @@ class Group {
   bool is_remote() const;
 
   /** Retrieves the query type. Errors if the group is not open. */
-  Status get_query_type(QueryType* query_type) const;
+  QueryType query_type_checked() const;
```

Comment:
The documentation should say what "checked" means. Putting `checked` in the name is inadequately descriptive and not necessary.